<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pokemonInfo</class>
 <widget class="QWidget" name="pokemonInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>445</width>
    <height>381</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>445</width>
    <height>381</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background/elements/pop.png);</string>
  </property>
  <widget class="QLabel" name="pokemonName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 700 28pt &quot;Hannotate SC&quot;;
border: none;</string>
   </property>
   <property name="text">
    <string>精灵名</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonImg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>141</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pokemon/elements/Pikachu.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonLvLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>LEVEL：</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonLv">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonHpLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>HP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonHp">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonAtk">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonAtkLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>ATK:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonDef">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonDefLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>DEF:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonExpLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>EXP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonExp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonSpeedLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>SPEED:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonSpeed">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonAbilityLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>技能：</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonAbilityName">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>技能名</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonAbilityExplain">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Wawati SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/elements/close.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
